src/components/PeopleFilters.tsx

```diff
@@ -87,7 +174,11 @@
       </div>
 
       <div className="panel-block">
-        <a className="button is-link is-outlined is-fullwidth" href="#/people">
+        <a
+          className="buttonis-linkis-outlinedis-fullwidth"
```

Comment:
There's a typo in the class name for the 'Reset all filters' button. It should be `className="button is-link is-outlined is-fullwidth"` to apply the correct styles.